analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CbdeHandler.CSharp10.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Sample
+    {
+        public void PositiveOverflow()
+        {
+            int i = 2147483600;
+            (i, int j) = (i + 100, 100); // FN
+        }
+
+        public void NegativeOverflow()
+        {
+            int i = -2147483600;
+            (i, int j) = (i - 100, 100); // FN
+        }
+
+    }
+
+    public struct ParameterlessConstructorAndFieldInitializer
+    {
+        int i;
+        public int Id { get; set; } = -2147483600;
```

Comment:
I don't know if CBDE supports interprocedural analysis. Can you please add also a test case with a field?